dam/loans/views.py

```diff
@@ -59,9 +60,10 @@ def allres(request):
 
 def allrets(request):
     rets = ItemLoan.objects.filter(returned_at__isnull=True)
-    args = {'returns': rets}
-    return render(request, 'loans/allReturns.html', args)
-
+    query = request.GET.get('q')
+    if query is not None:
+        rets = rets.filter(Q(item__name__icontains=query) | Q(item__description__icontains=query) | Q(client__first_name__icontains=query) | Q(client__last_name__icontains=query) | Q(client__email__icontains=query))
```

Comment:
Could you break this into multiple lines so this is easier to read (i.e. without needing to scroll as much)?
```suggestion
        rets = rets.filter(
            Q(item__name__icontains=query)
            | Q(item__description__icontains=query)
            | Q(client__first_name__icontains=query)
            | Q(client__last_name__icontains=query)
            | Q(client__email__icontains=query)
        )
```